index.html

```diff
@@ -1525,7 +1526,8 @@ <h2 id="vp-ld-json-jwt"><code>application/vp-ld+jwt</code></h2>
                 <code>application/vp-ld+jwt</code> Media Type specifically for
                 identifying a <a data-cite="JWT"></a>
                 with a payload
-                conforming to Verifiable Presentations.
+                conforming to the <a data-cite="VC-DATA-MODEL-2.0#verifiable-presentations">
+                Verifiable Presentation Data Model</a>.
```

Comment:
I'm not all that comfortable elevating "Verifiable Presentation Data Model" to a _thing_.  The phrase is only used once in VCDM and then it's not capitalized.

It's more wordy, but I'd happier with something like "conforming to the Verifiable Presentations definition in the Verifiable Credential Data Model.

Reply:
updated